core/deployment/src/main/java/io/quarkiverse/operatorsdk/deployment/OperatorSDKProcessor.java

```diff
@@ -261,6 +262,27 @@ void initializeRuntimeNamespacesFromBuildTimeValues(
 
     }
 
+    /**
+     * Ignore warnings related to non-indexed classes in the reflective hierarchy. At this point, we cannot know
+     * if they are actually needed for native compilation.
+     */
```

Comment:
Maybe add a comment that this might get removed once the underlying issue is addressed?

Reply:
Just for reference - https://github.com/quarkiverse/quarkus-operator-sdk/issues/941